src/ConcreteOperations/reflection.jl

```diff
@@ -0,0 +1,33 @@
+export reflect
+
+"""
+    reflect(P::LazySet{N}) where {N<:Real}
+
+Concrete reflection of a convex set `P`, resulting in the reflected set `-P`.
+
+### Note
+
+This function requires that the list of constraints of the set `P` is
+available.
+
+This function can be used to implement the alternative definition of the
+Minkowski Difference, which writes as
+```math
+A ⊖ B = \\{a − b | a ∈ A, b ∈ B} = A ⊕ (-B)
+P = \\{z ∈ ℝⁿ: sᵢᵀz ≤ rᵢ, i=1,...,N\\}.
+```
+by calling `minkowski_sum(A, reflect(B))`.
+"""
+function reflect(P::LazySet{N}) where {N<:Real}
```

Comment:
Just a comment: Since you do not use the `N` anywhere, you do not need to specify it.